### PR TITLE
add : added product list page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. added product list page 

snap shot 

image

### CODE REVIEW

File: components/product-page/product-page.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    justify-content: center;
+    margin-left:23rem;
+}
+
+.side-drawer{
+    position: fixed;
+    z-index:444;
+    background-color: var(--white-less);
+    height:100%;
+    font-size:1.2rem;
```
Comment (Sahil25061999) [inline components/product-page/product-page.css]: give header nav greater z-index value than side-drawer so the navbar shadow looks consistent across the page.

File: components/product-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+.main-body{
+    margin-top:7rem;
+}
+.header-nav{
+    position: fixed;
+    z-index:2;
+    top:0;
+    width:100%;
+    box-shadow: 0px 15px 10px -15px #111;
```
Comment (yaswanthmyneni) [inline components/product-page/product-page.css]: use css variables for colors.

**Review by Sahil25061999 — COMMENTED** [review]
cant see the sort by section of the filter, in terms of UI just a suggestion reduce the height of the navbar but its upto you.

File: components/product-page/product-page.css (hunk 2 above)
Comment (yaswanthmyneni) [reply components/product-page/product-page.css]: i cannot see complete aside bar.
Screenshot 2022-02-19 at 2 28 09 PM

**Comment by yaswanthmyneni** [comment]
you can merge after changes.

File: components/product-page/product-page.css (hunk 1 above)
Comment (JuvitaSaini09) [reply components/product-page/product-page.css]: done, i have made updated it ,please approve my PR

File: components/product-page/product-page.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/product-page/product-page.css]: I have made the changes,please approve my PR

**Review by yaswanthmyneni — APPROVED** [review]

**Review by Sahil25061999 — APPROVED** [review]

**Pull request merged**